code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -13,10 +13,7 @@
 	if(isliving(target))
 		var/mob/living/L = target
 		if(L.reagents)
-			L.reagents.add_reagent("toxin", poison_per_bite)
-			if(prob(poison_per_bite))
-				L << "<span class='danger'>You feel a tiny prick.</span>"

```

Comment:
you know whats also a fucking good warning that something bad might happen to you?
a giant fucking spider attacking you. the message is completely unnecessary and im not even removing it, it ALMOST NEVER OCCURED IN THE FIRST PLACE